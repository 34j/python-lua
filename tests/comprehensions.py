a = [i * j for i in range(5) for j in range(3) if i * j % 2 == 0 and i > 0 and j > 0]

for item in a:
    print(item)