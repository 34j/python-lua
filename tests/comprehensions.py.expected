2
2
4
6
6
4
8
12
10
1	1.0
2	4.0
3	9.0
4	16.0
5	25.0
